src/components/index.js

```diff
@@ -70,3 +70,4 @@ export { default as TextWithHighlight } from "./TextWithHighlight/TextWithHighli
 export { default as Accordion } from "./Accordion/Accordion/Accordion";
 export { default as AccordionItem } from "./Accordion/AccordionItem/AccordionItem";
 export { default as Clickable } from "./Clickable/Clickable";
+export * from "../utils/colors-utils";
```

Comment:
its not a component and therefore I think you can remove this

Reply:
@orrgottlieb wdyt?

Reply:
in the future we will create separate index files for hooks, utils and components